Lib/test/libregrtest/worker.py

```diff
@@ -14,9 +14,9 @@
 
 
 USE_PROCESS_GROUP = (hasattr(os, "setsid") and hasattr(os, "killpg"))
-NEED_TTY = set('''
-    test_ioctl
-'''.split())
+NEED_TTY = {
```

Comment:
Not related.

Reply:
Yeah, it's a follow-up of a previous PR. I didn't want to write a dedicated PR for that.